src/components/atoms/button/button.scss

```
@@ -1,11 +1,56 @@
 .button {
   padding: $medium-space $xxx-large-space;
   border-radius: $regular-radius;
+  cursor: pointer;
   border: none;
 
```

je pense qu'il manque un 
```suggestion
border: 2px solid transparent;
```
si tu vas dans le formulaire, les boutons "sautent" quand tu les hover parce qu'ils prennent une border d'un coup. Du coup pas besoin de le rajouter dans &--primary (suggestion thomas) parce qu'ils en ont tous besoin